content/desktop/settings/windows.md

```diff
@@ -216,6 +212,25 @@ The HTTPS proxy settings used for scanning images are set using the `HTTPS_PROXY
 If you are running Windows containers in Docker, you can allow the Windows Docker daemon to use Docker Desktop's internal proxy, with the **Use proxy for Windows Docker daemon** setting.
 This is useful when a corporate proxy that requires authentication is manually configured or set at the system level. If you are an admin for your organization and have a Docker Business subscription, you can control this setting with [Settings management](../hardened-desktop/settings-management/configure.md) using the `windowsDockerdPort` parameter.
 
+#### Proxy authentication
+
+If the proxy requires authorization then Docker Desktop dynamically asks
+the developer for a username and password. All passwords are stored securely in the OS credential store.
```

Comment:
```suggestion
If the proxy requires authentication then Docker Desktop dynamically asks
the developer for a username and password. All passwords are stored securely in the OS credential store.
```

I also wonder about the phrase "dynamically asks/prompts", here and in the basic auth section below. I guess it would be better to say that basic auth triggers an authentication prompt (and how/when it's triggered)